clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -3477,7 +3481,6 @@ static const char *paramKind2Str(KernelParamKind K) {
     CASE(std_layout);
     CASE(sampler);
     CASE(pointer);
-  default:
     return "<ERROR>";
```

Comment:
Shouldn't the return "<ERROR>" statement be moved outside the switch block?

Reply:
Good catch! Updated it. Thanks.